translations/messages.fr.yml

```
@@ -44,19 +44,29 @@ forms:
             placeholder: "Exemple : Botaniste"
         bio:
             label: "Bio"
-            hint: "Précisez ce que vous faites dans la vie, ce que vous recherchez sur la plateforme… (sera affiché sur votre profil)"
+            hint: "Indiquez votre niveau de connaissance sur les pollinisateurs, ce qui vous intéresse, les actions que vous effectuez ou recherchez, etc. Tout ce que vous souhaitez partager avec les autres (sera affiché sur votre profil)."
             placeholder: "Dites nous en plus sur vous…"
         inscription_type:
             label: "Type d’inscription"
             labels:
                 private: "Inscription à titre personnel"
                 professional: "Inscription à titre professionnel"
+        favorite_environment:
+            label: "Milieu d'action"
```

Bien utiliser `’` au lieu de `'` dans les messages
```suggestion
            label: "Milieu d’action"
```